tests/Unit/ControlSystem/Test_FunctionOfTime.cpp

```diff
@@ -0,0 +1,88 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#include <catch.hpp>
+
+#include "ControlSystem/PiecewisePolynomial.hpp"
+#include "Utilities/ConstantExpressions.hpp"
+#include "tests/Unit/TestHelpers.hpp"
+
+SPECTRE_TEST_CASE("Unit.ControlSystem.FunctionOfTime.PiecewisePolynomial",
```

Comment:
Just replace the PiecewisePolynomial tests that are identical to the ones here with these ones. Basically, you can combine them into one thing that tests that PiecewisePolynomial properly derives off of the base class.